zebra-state/src/response.rs

```diff
@@ -59,6 +59,10 @@ pub enum Response {
     ///
     /// Does not check transparent UTXO inputs
     ValidBestChainTipNullifiersAndAnchors,
+
+    #[cfg(feature = "getblocktemplate-rpcs")]
+    /// Response to [`Request::CheckBlockValidity`](crate::Request::CheckBlockValidity)
+    ValidBlock(block::Hash),
```

Comment:
Is there a way to make it clear that this is not actually guaranteed to be a valid block?
We might also want to update the documentation on this response variant.

```rust
    ValidBlockProposal(block::Hash),
```

Reply:
Renamed in [`8420b75`](https://github.com/ZcashFoundation/zebra/pull/5870/commits/8420b75013c1a05ea11a9799d1f87bd338c3c27c).